### PR TITLE
🍒 Multichain tutorial

### PR DESCRIPTION
**Pull request by yivlad**

### CODE REVIEW

File: packages/docs/docs/tutorials/multichain.mdx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+## Prerequisites
+
+The tutorial assumes the user has already started with the basics of `useDApp`.
+
+See the [Getting Started](../getting-started.mdx) guide if you are a new user.
+
+## Motivation
+
+The most common approach for a DApp is to operate on a single chain. But there are some use cases when one would want to display or/and modify data on multiple chains simultaneously.
```
Comment (yivlad) [inline packages/docs/docs/tutorials/multichain.mdx:16]:
```suggestion
The most common approach for a DApp is to operate on a single chain. But there are some use cases when one would want to display data from multiple chains simultaneously. 
```

File: packages/docs/docs/tutorials/multichain.mdx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+The tutorial assumes the user has already started with the basics of `useDApp`.
+
+See the [Getting Started](../getting-started.mdx) guide if you are a new user.
+
+## Motivation
+
+The most common approach for a DApp is to operate on a single chain. But there are some use cases when one would want to display or/and modify data on multiple chains simultaneously. 
+
+`useDApp` has inbuilt multichain support.
```
Comment (yivlad) [inline packages/docs/docs/tutorials/multichain.mdx:18]:
```suggestion
`useDApp` has built-in multichain support.
```

File: packages/docs/docs/tutorials/multichain.mdx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+In this tutorial we will create a simple DApp that will display simultaneously balance of some address on Mainnet and balance of another address on Kovan.
+
+## Add Kovan network to config
+
+Please make the following changes to the app's entrypoint file.
```
Comment (yivlad) [inline packages/docs/docs/tutorials/multichain.mdx:24]: Add explanation

File: packages/docs/docs/tutorials/multichain.mdx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+See the [Getting Started](../getting-started.mdx) guide if you are a new user.
+
+## Motivation
+
+The most common approach for a DApp is to operate on a single chain. But there are some use cases when one would want to display data from multiple chains simultaneously. 
+
+`useDApp` has built-in multichain support.
+
+In this tutorial we will create a simple DApp that will display simultaneously balance of the address both on Mainnet and Kovan.
```
Comment (rzadp) [inline packages/docs/docs/tutorials/multichain.mdx:20]:
```suggestion
In this tutorial we will create a simple DApp that will display simultaneously balance of an address both on Mainnet and on Kovan.
```

File: packages/docs/docs/tutorials/multichain.mdx [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+## Add Kovan network to config
```
Comment (rzadp) [inline packages/docs/docs/tutorials/multichain.mdx:22]:
```suggestion
## Add Kovan network to the config
```

File: packages/docs/docs/tutorials/multichain.mdx [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+As a first step we need to add the Kovan network configuration. As Kovan is a network supported by default in `useDApp`, the only thing we have to do is to specify an endpoint where requests can be sent to. In our case we will use Infura - free Etherum API, that also supports the Kovan testnet. 
+Please make the following changes to the app's entrypoint file. 
+````diff
+  import {
+    Mainnet,
+    Kovan,
```
Comment (rzadp) [inline packages/docs/docs/tutorials/multichain.mdx:29]:
```suggestion
+   Kovan,
```

File: packages/docs/docs/tutorials/multichain.mdx [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+    <React.StrictMode>
+      <DAppProvider config={config}>
+        <App />
+      </DAppProvider>
+    </React.StrictMode>,
+    document.getElementById('root')
+  )
+````
+
+## Display balances on Mainnet and Kovan
```
Comment (rzadp) [inline packages/docs/docs/tutorials/multichain.mdx:57]:
```suggestion
## Display balances on Mainnet and on Kovan
```

(just for consideration, it sounds better to me)

File: packages/docs/docs/tutorials/multichain.mdx [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+
+Our app will be based on a one described in the [Getting Started](../getting-started.mdx) guide.
```
Comment (rzadp) [inline packages/docs/docs/tutorials/multichain.mdx:59]:
```suggestion
Our app will be based on one described in the [Getting Started](../getting-started.mdx) guide.
```

**Review by rzadp — COMMENTED** [review]
Consider adding a wrapping up summary of what we did, and maybe how it works.

**Review by rzadp — APPROVED** [review]

**Pull request merged**